data/meson.build

```diff
@@ -57,3 +59,43 @@ mode_dir = join_paths(data_dir, 'gnome-shell', 'modes')
 install_data('initial-setup.json', install_dir: mode_dir)
 
 install_data('gnome-welcome-tour', install_dir: get_option('libexecdir'))
+
+eos_demo_mode_data_dir = join_paths(get_option('datadir'), 'eos-demo-mode')
+install_data('eos-demo-video-C.webm', install_dir: eos_demo_mode_data_dir)
+install_data('eos-demo-video-id.webm', install_dir: eos_demo_mode_data_dir)
+
+policykit_actions_dir = join_paths(data_dir, 'polkit-1', 'actions')
+
+configure_file(
+    input: 'com.endlessm.TestMode.policy.in',
+    output: 'com.endlessm.TestMode.policy',
+    install: true,
+    install_dir: policykit_actions_dir,
+    configuration : conf
```

Comment:
If you can be bothered: the space before colon (here and in the next block) is inconsistent with all other params.

Reply:
For sure.

Reply:
It's still inconsistent!